source/StartConditions.cpp

```diff
@@ -47,9 +62,22 @@ void StartConditions::Load(const DataNode &node)
 			else
 				child.PrintTrace("Skipping unsupported use of a \"stock\" ship (a full definition is required):");
 		}
+		else if(child.Token(0) == "conversation" && child.HasChildren())
+			conversation.Load(child);
+		else if(child.Token(0) == "conversation" && child.Size() >= 2)
+			stockConversation = GameData::Conversations().Get(child.Token(1));
 		else
 			conditions.Add(child);
 	}
+	if(name.empty())
+	{
+		if (node.Size() >= 2)
+			name = node.Token(1);
+		else
+			name = "Unnamed start";
+	}
```

Comment:
This should be done before parsing the children. The unique name is overwritten by the base name, not the other way around.